README.md

```diff
@@ -105,3 +105,33 @@ Or.
 4. Install ninja: `brew install ninja`
 6. Run `./build.sh`
 7. The resulting binary will be in build/release
+
+## License
+
+Leela Chess is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Leela Chess is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Leela Chess.  If not, see <http://www.gnu.org/licenses/>.
+
+### Additional permission under GNU GPL version 3 section 7
+
+_The source files of Lc0 with the exception of the BLAS and OpenCL
+backends (namely `network_blas.cc`, `network_opencl.cc` and all files
+in the `blas` and `opencl` directories have the following additional
+permission as allowed under GNU GPL version 3 section 7:_
```

Comment:
I think we should isolate those files, at least put them into a separate directory, with it's own COPYING and README (which i.e. reminds that it's not allowed to copy-paste the code from them into other sources, and is discouraged to edit files of different licenses in one PR).

Probably we can do that later, but we should not forget.